src/Components/Home.js

```diff
@@ -22,6 +24,18 @@ function Home() {
   return (
     <main>
       {showProgress && <ProgressBar mode="indeterminate" />}
+      {skeleton
+        ? <div>
+        <div style={{ display: 'none' }}>
+      {setTimeout(() => {
+        setskeleton(false)
+      }, 3000)}
```

Comment:
Can this be moved outside the html?

Reply:
3000ms (3s) is too long, I think 300ms is enough

Reply:
I think 350ms would be better

Reply:
300ms would be enought!